src/components/Image/src/Image.vue

```diff
@@ -109,6 +109,11 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		objectFit: {
+			type: String,
+			validator: (fit) => ['cover', 'contain', 'scale-down'].includes(fit),
```

Comment:
We can probably import cssValidator here to allow all valid values for `'object-fit'.
https://github.com/square/maker/blob/master/src/components/Text/src/Text.vue#L48